pfcpiface/bess.go

```diff
@@ -342,6 +346,44 @@ func (b *bess) SummaryLatencyJitter(uc *upfCollector, ch chan<- prometheus.Metri
 	measureIface("Core", uc.upf.coreIface)
 }
 
+func (b *bess) SummaryGtpuLatency(uc *upfCollector, ch chan<- prometheus.Metric) {
```

Comment:
Would the call to this function be successful if path monitoring is disabled?

Reply:
The behavior is the same as when the `metrics` flag is off

Here is how the pipeline looks like with the "enable_gtpu_path_monitoring` flag `on` (true)
![image](https://github.com/omec-project/upf/assets/87488727/516623e2-6d74-4097-bb43-9b37248d1a1d)

And here is how the pipeline looks like with the "enable_gtpu_path_monitoring` flag `off` (false)
![image](https://github.com/omec-project/upf/assets/87488727/b783134b-c67c-40d3-a42b-d606180df85f)